pkg/kind/kind.go

```diff
@@ -132,7 +159,7 @@ func createLocalRegistry() error {
 }
 
 func connectLocalRegistry() error {
-	connectLocalRegistry := exec.Command("docker", "network", "connect", "kind", container_reg_name)
+	connectLocalRegistry := exec.Command(container_engine, "network", "connect", "kind", container_reg_name)
 	if err := connectLocalRegistry.Run(); err != nil {
 		return fmt.Errorf("failed to connect local registry to kind cluster")
```

Comment:
This would be a nice to have :smile: 

```suggestion
		return fmt.Errorf("failed to connect local registry to kind cluster: %w", err)
```